### PR TITLE
[JOSE-02] Fix emit behavior and add list component features

### PR DESCRIPTION
**Pull request by josepenagosGL**
Hi team, 

just fixed the emit behavior (both in search component and search lightbox) and added some tweaks to list component.

### CODE REVIEW

**Review by ogranada — APPROVED** [review]
We need to discuss about the mock flag in order to create a global way to enable it.
Looks good to me 👍 ⚡️

File: src/views/Home.vue [hunk 1]
```diff
@@ -20,11 +20,18 @@ export default {
...
+        this.podcasts = res;
+      });
+    },
+  },
   components: {
     ListPodcast,
   },
   created() {
-    api.search('software').then((res) => {
+    api.search('software', true).then((res) => {
```
Comment (alejandrobenjumeaGL) [inline src/views/Home.vue]: Please remove mock boolean

File: src/views/Home.vue [hunk 2]
```diff
@@ -20,11 +20,18 @@ export default {
       podcasts: [],
     };
   },
+  methods: {
+    filterPodcasts(term) {
+      api.search(term, true).then((res) => {
```
Comment (alejandrobenjumeaGL) [inline src/views/Home.vue]: Please remove mock boolean

File: src/components/search-lightbox/search-lightbox.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <search-input
-      :change="searchTracks"
+      @search-track="searchTracks"
```
Comment (alejandrobenjumeaGL) [inline src/components/search-lightbox/search-lightbox.vue]: I think 'search-track' is defining a specific scope. But this is a generic search component. So I suggest rename it to **change**

**Review by alejandrobenjumeaGL — CHANGES_REQUESTED** [review]

**Review by alejandrobenjumeaGL — APPROVED** [review]

**Pull request merged**